app.js

```diff
@@ -33,11 +33,14 @@ app.use((err, req, res, next) => {
 });
 
 app.use((err, req, res, next) => {
-  if (err.code === '22P02')
-    res.status(400).send({ msg: 'Bad request, invalid article_id' });
-  else if(err.code ==='23503')
-    res.status(400).send({ msg: 'Bad request, user doesn\'t exist'})
-  else {
+  if (err.code === '22P02'){
+    res.status(400).send({ msg: 'Bad request, invalid article_id' })
+  } else if(err.code ==='23503') {
+    res.status(400).send({ msg: 'Bad request'})
+
+  } else if(err.code === '23502'){
+    res.status(400).send({msg: 'Bad request, missing information'})
+} else {
```

Comment:
Good error handling here, something to think about in the future is that you could clean this up a bit by using || in the if statement or even placing the error codes in an array.